generatebundlefile/promote.go

```diff
@@ -207,6 +220,19 @@ func (c *SDKClients) PromoteHelmChart(repository, authFile string, crossAccount
 			}
 		}
 	}
+
+	// Remove this once we add support for Private ECR pull in the Helm Driver.
+	// This will hard force the Helm chart to be moved from Private ECR to Public ECR if it didn't exist.
+	for _, images := range helmRequires.Spec.Images {
+		source := fmt.Sprintf("docker://%s.dkr.ecr.us-west-2.amazonaws.com/%s:%s", c.stsClient.AccountID, images.Repository, images.Tag)
+		destination := fmt.Sprintf("docker://%s/%s:%s", c.ecrPublicClient.SourceRegistry, images.Repository, images.Tag)
+		BundleLog.Info("Copying Helm Digest, and Tag to destination location......", "Location", fmt.Sprintf("%s/%s:%s %s", c.ecrPublicClient.SourceRegistry, images.Repository, images.Tag, images.Digest))
+		err = copyImage(BundleLog, authFile, source, destination)
+		if err != nil {
+			return fmt.Errorf("Unable to copy image from source to destination repo %s", err)
+		}
+		continue
```

Comment:
What does continue do here? Wouldn't it loop back naturally anyway?

Reply:
yea technically it's not needed here.

Reply:
I'll take it out.

Reply:
Good catch.